nf-test.config

```
@@ -12,10 +12,10 @@ config {
     ignore 'modules/nf-core/**/tests/*', 'subworkflows/nf-core/**/tests/*'
 
     // run all test with defined profile(s) from the main nextflow.config
-    profile "test"
+    profile "test_taxid"
 
```

```suggestion
    profile "test_taxid"
    options "-dump-channels"
```
for runner debugging purposes, because it seems fine locally